src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+    public static void main(String[] args) {
+        Plane boeing747 = new Plane.PlaneBuilder().setModel("Boeing 747")
+                .setColor("Red with white stripes").setYear(1990)
+                .setPassengersCapacity(452).setEngineThrust(22.6).build();
```

Comment:
1)Don't use numbers in variable namings;
2)Let's call each method of the builder from the new line;